### DISCUSSION TITLE
Feature Requests and maybe an issue.

### DISCUSSION THREAD

**Discussion by everythingability** [hiukim/mind-ar-js#98] [Ideas]
Hi, love the project, just tried it ... amazingly well done. Apologies for posting here... 

Feature Request #1: Can you add opacity on videos please? So they "bleed" through the image, sorta.
Feature Request #2: Can the QR code downloaded use the same name as the project? It'll make managing a large number of them easier.

Issue (maybe)
The "fixing" to position of the trigger image seems a little behind the times, is that just because this is browser-based? By that when I move my phone, the video moves too, rather than it "sticking" to the trigger... if this is just a side-effect, please ignore, this is exactly the kind of tool I use/want to use with students. FAB!

Thank you.

**Comment by everythingability** [comment]
Oh hang on, maybe I can do opacity with the scripting element. THAT'S COOL TOO.

**Comment by hiukim** [comment]
@everythingability Thanks. I'm glad if it could help with your teaching. I'm also interested to do some workshops in the future, maybe for high school students to try building simple web AR applications. Would definitely love to hear your experiences and feedback.

I guess you are referring to the Pictarize platform.  Will definitely consider those suggestions.

For the "fixing" issue, the augmented content (i.e. video in your case) should sticked to the target image. If it's off or delayed, probably just due to tracking instability.
